.chronus/changes/fix_override-2024-10-13-15-32-53.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: fix
+packages:
+  - "@azure-tools/typespec-client-generator-core"
+---
+
+do not replace operation with `@override` when `listOperationsInOperationGroup`
```

Comment:
Does it only affect `listOperationsInOperationGroup`?

Java emitter currently list `SdkServiceMethod` under `client.methods`.

From the context, this seems also should not be the `@override` operation -- it should have the signature of that in `SdkServiceMethod`, while having the signature of the original/overrided operation in `SdkServiceOperation`

Reply:
it is just a bug for previous `@override` impl. for `SdkServiceMethod` it will use the `@override` operation, for `SdkServiceOperation` it will use the original operation, these two logic is right for the impl.